client/ionic/src/pages/Triage.tsx

```diff
@@ -14,6 +14,19 @@ class Triage extends React.Component<{}, TriageState> {
   constructor(args: Readonly<{}>) {
     super(args);
     const survey = new Survey.ReactSurveyModel(triage);
+
+    // Update CSS classes
+    survey.onUpdateQuestionCssClasses.add((survey, options) => {
+      const classes = options.cssClasses;
+      classes.mainRoot += ' f6';
+      classes.root = 'sq-root';
+      classes.title += '';
```

Comment:
Nit: Any reason for these empty string lines?

Reply:
Left this as an issue for now Styling the Triage Form #178

This is still a work in progress, I ran out of time